src/query/api/v1/handler/prometheus/native/read.go

```diff
@@ -84,13 +85,26 @@ func NewPromReadHandler(
 }
 
 func (h *PromReadHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	result, params, err := h.ServeHTTPWithEngine(w, r, h.engine)
+	if err != nil {
+		return
+	}
+
+	// TODO: Support multiple result types
+	w.Header().Set("Content-Type", "application/json")
+	w.Header().Set("Access-Control-Allow-Origin", "*")
```

Comment:
Don't think this is necessary

Reply:
Correct; `CORS` now gets set by middleware. Content-Type is still necessary though (we should probably factor that out too).